tests/cli/test_subcommands.py

```diff
@@ -188,7 +188,7 @@ def test_update(subcommand: str, conda_cli: CondaCLIFixture, tmp_env: TmpEnvFixt
 
 
 def test_env_list(conda_cli: CondaCLIFixture):
-    assert conda_cli("env", "list") == conda_cli("info", "--envs")
+    assert conda_cli("env", "list")[0] == conda_cli("info", "--envs")[0]
```

Comment:
Do we need this? stdout, stderr, and exitcode should be identical

Reply:
[This line](https://github.com/conda/conda/pull/13173/files#diff-b3cd58236ff4632f94ca40c54521052d1211fa67694c0e3e71b4f7c26c055dc3R44) was missing due to a merge conflict that left it out by accident. Will revert this test to how it was previously!